rimage/transform/pinhole_camera_parameters_test.go

```diff
@@ -204,3 +207,16 @@ func TestGetCameraMatrix(t *testing.T) {
 	test.That(t, intrinsicsK.At(1, 2), test.ShouldEqual, intrinsics.Ppy)
 	test.That(t, intrinsicsK.At(2, 2), test.ShouldEqual, 1)
 }
+
+func TestNilIntrinsics(t *testing.T) {
+	var nilIntrinsics *PinholeCameraIntrinsics
+
+	// should not panic
+	nilIntrinsics.CheckValid()
+	nilIntrinsics.GetCameraMatrix()
+	nilIntrinsics.PixelToPoint(0.0, 0.0, 0.0)
+	nilIntrinsics.PointToPixel(0.0, 0.0, 0.0)
+	nilIntrinsics.ImagePointTo3DPoint(image.Point{}, rimage.Depth(0))
+	nilIntrinsics.RGBDToPointCloud(&rimage.Image{}, &rimage.DepthMap{})
+	nilIntrinsics.PointCloudToRGBD(pointcloud.PointCloud(nil))
```

Comment:
You can use test.ShouldNotPanic so that it doesn't actually panic and cause the testing program to crash

You need to wrap them each in a function i.e.
```
test.That(t, func() {  nilIntrinsics.CheckValid() }, test.ShouldNotPanic)
```

Reply:
Oh fancy! Done.